src/security/pci-guidance.md

```diff
@@ -0,0 +1,151 @@
+# PCI DSS Responsibility Guidance
+
+Platform.sh has many PCI certified customers using our services. Below we will attempt to clarify the PCI compliance responsibilities for Platform.sh and our customers.
+
+Simple audit questions can be handled via support tickets however in-depth support for your audit may required additional arrangements. Please contact your Platform.sh account manager to discuss your needs.
+
+
+## Overview
+
+Platform.sh provides a Platform as a Service (PaaS) solution which our customers may use to store, process, or transmit cardholder data.
+
+
+## Security & Compensating Controls
+
+ * For those customers needing to meet SAQ C or SAQ D, a Platform.sh Enterprise Plan may be needed in order to fully meet your needs. Please [contact us](https://platform.sh/contact) for more information.
+
+ * For a list of security measures, please see our [Security page](https://platform.sh/security).
+
+ * Please take note that customer environments are deployed in a read-only instance, segregated with GRE and IPSEC tunnels, which often permits compensating controls to be claimed for several PCI requirements.
+
+ * Because customers can use our PaaS in a variety of ways, the best approach with auditors is to focus is on “What do I, the customer, control/configure and how is it managed in a compliant manner?”
+
+
+## Responsibility
+
+Platform.sh customers are ultimately responsible for their own PCI DSS compliance.
+
+The following sections provide guidance on shared responsibilities to achieve PCI DSS compliance. We have chosen to list all the sections found in a typical SAQ D although you may not need this level of detail for your efforts. This section is currently using PCI DSS 3.2 as a reference.
```

Comment:
Please reword this to avoid the need for the royal we if you can.  Like, "This list below covers all sections found..."